packages/test-peerpods.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+# Copyright 2024 Edgeless Systems GmbH
+# SPDX-License-Identifier: AGPL-3.0-only
+
+set -euo pipefail
+
+# TODO(burgerdev): re-enable image upload once rest is working
+true || nix run -L .#scripts.upload-image -- \
+  --subscription-id="${azure_subscription_id:?}" \
+  --location="${azure_location:?}" \
+  --resource-group="${azure_resource_group:?}"
+image_id=/subscriptions/0d202bbb-4fa7-4af8-8125-58c269a05435/resourceGroups/contrast-ci/providers/Microsoft.Compute/galleries/contrast_ci_contrast/images/contrast/versions/0.0.1730981348
+echo "image_id = \"${image_id}\"" > infra/azure-peerpods/image_id.auto.tfvars
+
+cat >infra/azure-peerpods/testrun.auto.tfvars <<EOF
+name_prefix = "${azure_resource_group:?}"
+resource_group = "${azure_resource_group:?}"
+subscription_id = "${azure_subscription_id:?}"
+tenant_id = "${azure_tenant_id:?}"
+client_id = "${azure_client_id:?}"
+client_secret = "${azure_client_secret:?}"
```

Comment:
Should these be `justfile.env` vars?

Reply:
Where will these come from when doing a just create as developer?

Reply:
Well, you can create an app etc. for your own testing. This may be needed often enough to add a target for that, wdyt? Essentially, the part I'm currently removing from Terraform would go into its own module.

Reply:
Yes, we should add a separate target/module for this.

Reply:
Added a module